migrations/dev-20210610-add-users-changelogViewDate-column.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, DataTypes) => {
+    await queryInterface.addColumn('Users', 'changelogViewDate', {
```

Comment:
I find it a bit overkill to add a plain SQL column to store this. Could have been stored in `data` at this stage.

Personal opinion, not asking for a change for now.